tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/contact_call_to_action.yaml

```diff
@@ -1,21 +1,23 @@
 context:
   value:
-    text: 'Work with the best in the business'
-    button_text: 'Contact Us'
-    button_link:
-      - internal_link:
-          id: 1
-      - external_link:
-          url: 'https://example.com'
-      - email:
-          email: 'example@example.com'
+    call_to_action:
+      - value:
+          text: Get in touch to learn about our journey to becoming employee-owned
+          button_text: 'Contact Us'
+          button_link: True
     person:
-      name: 'John Doe'
-      role: 'CEO'
+      name: Olly Willans
+      role: Chief People and Planet Officer
 
 tags:
   image:
-    avatar width-143 format-webp class="avatar__image":
-      # target_var: person_image
-      raw: |
-        <img src="https://source.unsplash.com/143x143?face" width="143" height="143" alt="Some alt" class="avatar__image">
+    avatar fill-144x144 format-webp as avatar_desktop:
```

Comment:
Do you think it is worth adding a different image variant for the small avatar at desktop? I think perhaps the extra code needed isn't worth it but interested if you had any thoughts.

Reply:
Hmm, that's a good question. I think i'd agree that the extra code probably isn't worth it as the image we're serving it so small. Maybe a nice to have if we get time for it : )